packages/codemirror-extension/src/index.ts

```diff
@@ -332,12 +332,15 @@ function activateEditorCommands(
     'the-matrix',
     'xq-light',
     'zenburn'
-  ].forEach(name =>
+  ].forEach(name => {
+    if (name === 'codemirror') {
```

Comment:
Thanks @joker2600, it looks like this doesn't actually change the label in the menu. I think we need to change it up above in the definition for the `changeTheme` command. On line 208 the label is computed from the args given here (which end up being `'default'`).